folding/src/error_term.rs

```diff
@@ -187,13 +187,22 @@ pub(crate) fn compute_error<C: FoldingConfig>(
     let u_cross = ul * ur;
     let zero = || EvalLeaf::Result(env.inner().zero_vec(env.domain.size as usize));
 
+    let alphas_l = env
+        .get_relaxed_instance(Side::Left)
+        .inner_instance()
+        .get_alphas();
+    let alphas_r = env
+        .get_relaxed_instance(Side::Right)
+        .inner_instance()
+        .get_alphas();
+
     let t_0 = {
         let t_0 = (zero(), zero());
         let (l, r) = exp.degree_0.iter().fold(t_0, |(l, r), (exp, sign, alpha)| {
             //could be left or right, doesn't matter for constant terms
             let exp = eval_exp_error(exp, env, Side::Left);
-            let alpha_l = env.inner().alpha(*alpha, Side::Left);
-            let alpha_r = env.inner().alpha(*alpha, Side::Right);
+            let alpha_l = alphas_l.get(*alpha).expect("alpha not present");
```

Comment:
So now alphas are part of a relaxed instances as opposed to the environment. Makes sense.

Reply:
They have always been part of a relaxed instance.